### PR TITLE
[dom] Add recipe for singularity 3.8.0-daint

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

File: easybuild/easyconfigs/s/singularity/singularity-3.8.0-daint.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'https://www.sylabs.io/'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = SYSTEM
+
+modtclfooter = """
+# Swap the programming environment
+module unload PrgEnv-cray
+module unload PrgEnv-intel
+module unload PrgEnv-pgi
```
Comment (teojgo) [inline easybuild/easyconfigs/s/singularity/singularity-3.8.0-daint.eb:17]: @victorusu do you think that PrgEnv-nvidia should also be added here?

Comment (victorusu) [reply easybuild/easyconfigs/s/singularity/singularity-3.8.0-daint.eb:17]: Yes

**Review by victorusu — APPROVED** [review]
`lgtm`

**Comment by lucamar** [comment]
@teojgo Is this pull request ready to be merged or do you need to implement the change that you asked @victorusu?

**Comment by teojgo** [comment]
> @teojgo Is this pull request ready to be merged or do you need to implement the change that you asked @victorusu?

I will implement the change first.

**Pull request merged**